glean-core/python/tests/conftest.py

```diff
@@ -4,25 +4,21 @@
 
 
 import logging
+import os
 import socket
 import time
 
-
 import pytest
 
-
 from glean import testing
 from glean import __version__ as glean_version
 
-# !IMPORTANT!
-# Everytime this hash is changed it should also be changed in
-# glean-core/android/build.gradle
-GLEAN_PING_SCHEMA_GIT_HASH = "63dcb4285b73c0c625cbee46cf1fe506b7f4c5f6"
-GLEAN_PING_SCHEMA_URL = (
-    "https://raw.githubusercontent.com/mozilla-services/"
-    "mozilla-pipeline-schemas/{}/schemas/glean/glean/"
-    "glean.1.schema.json"
-).format(GLEAN_PING_SCHEMA_GIT_HASH)
+# This defines the location of the JSON schema used to validate the pings
+# created during unit testing. This uses the vendored schema.
+#
+# Use `bin/update-schema.sh latest` to update it to the latest upstream version.`
+this_dir = os.path.dirname(__file__)
+GLEAN_PING_SCHEMA_PATH = os.path.join(this_dir, "../../../glean.1.schema.json")
```

Comment:
Let's use [pathlib](https://docs.python.org/3/library/pathlib.html) to be consistent with the rest of the Glean Python bindings:

At top:

```
from pathlib import Path
```

```suggestion
this_dir = Path(__file__.parent)
GLEAN_PING_SCHEMA_PATH = path.parent.parent.parent / "glean.1.schema.json"
```

Reply:
`parent.parent.parent` - lol

Reply:
Where is the `greatgrandparent` property?

Reply:
don't make me implement that!